db/schema.rb

```diff
@@ -1147,10 +1147,8 @@
     t.string "address_line1"
     t.string "address_line2"
     t.string "address_line3"
-    t.string "representative_number"
     t.index ["location"], name: "index_veteran_organizations_on_location", using: :gist
     t.index ["name"], name: "index_veteran_organizations_on_name"
-    t.index ["poa", "representative_number"], name: "index_veteran_organizations_on_poa_and_representative_number", unique: true
```

Comment:
I don't know if relying on the automatic index removal is rollback safe meaning if the migration was rolled back, I don't think the index would get re-added. We don't intend to ever use the column or the index, so it's probably not a big deal. Just wanted to call this out in case you had more information on the behavior.

Reply:
I tried explicitly removing the index but got an error:
```
class RemoveRepresentativeNumberFromVeteranOrganizations < ActiveRecord::Migration[6.1]
  disable_ddl_transaction!

  def change
    remove_index :veteran_organizations, column: [:poa, :representative_number], algorithm: :concurrently
    remove_column :veteran_organizations, :representative_number
  end
end
```

```
-- remove_index(:veteran_organizations, {:column=>[:poa, :representative_number], :algorithm=>:concurrently})
rails aborted!
StandardError: An error has occurred, all later migrations canceled: (StandardError)

No indexes found on veteran_organizations with the options provided.
/Users/holdenhinkle/Documents/Oddball/va_website/vets-api/db/migrate/20231201160018_remove_column_and_index_from_veteran_organizations_table.rb:5:in `change'

Caused by:
ArgumentError: No indexes found on veteran_organizations with the options provided. (ArgumentError)

            raise ArgumentError, "No indexes found on #{table_name} with the options provided."
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
/Users/holdenhinkle/Documents/Oddball/va_website/vets-api/db/migrate/20231201160018_remove_column_and_index_from_veteran_organizations_table.rb:5:in `change'
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

Then I inspected the table:
```
vets-api-# \d veteran_organizations
                           Table "public.veteran_organizations"
          Column           |            Type             | Collation | Nullable | Default 
---------------------------+-----------------------------+-----------+----------+---------
 poa                       | character varying(3)        |           |          | 
 name                      | character varying           |           |          | 
 phone                     | character varying           |           |          | 
 state                     | character varying(2)        |           |          | 
 created_at                | timestamp without time zone |           | not null | 
 updated_at                | timestamp without time zone |           | not null | 
 address_type              | character varying           |           |          | 
 city                      | character varying           |           |          | 
 country_code_iso3         | character varying           |           |          | 
 country_name              | character varying           |           |          | 
 county_name               | character varying           |           |          | 
 county_code               | character varying           |           |          | 
 international_postal_code | character varying           |           |          | 
 province                  | character varying           |           |          | 
 state_code                | character varying           |           |          | 
 zip_code                  | character varying           |           |          | 
 zip_suffix                | character varying           |           |          | 
 lat                       | double precision            |           |          | 
 long                      | double precision            |           |          | 
 location                  | geography(Point,4326)       |           |          | 
 raw_address               | jsonb                       |           |          | 
 address_line1             | character varying           |           |          | 
 address_line2             | character varying           |           |          | 
 address_line3             | character varying           |           |          | 
 representative_number     | character varying           |           |          | 
Indexes:
    "index_veteran_organizations_on_location" gist (location)
    "index_veteran_organizations_on_name" btree (name)
    "index_veteran_organizations_on_poa" UNIQUE, btree (poa)

vets-api-# 
```

There index wasn’t there.

That's why I didn't explicitly remove it in this PR.